src/app/pages/PerpetualPage/components/OrderHistoryTable/OrderHistoryRow.tsx

```diff
@@ -36,14 +37,6 @@ type OrderHistoryRowProps = {
 export const OrderHistoryRow: React.FC<OrderHistoryRowProps> = ({ item }) => {
   const { t } = useTranslation();
 
```

Comment:
```suggestion
  const {
    position,
    tradeType,
    datetime,
    pair,
    execSize,
    execPrice,
    orderId,
  } = item;
```

Reply:
I would use object destructuring at the beginning and then use the variables so you don't always have to write `item.something`.